rpc/network/network.go

```diff
@@ -140,7 +148,7 @@ func (nm *Manager) Init(networks []params.Network) error {
 						errors += fmt.Sprintf("error updating network fallback_url for ChainID: %d, %s", currentNetworks[j].ChainID, err.Error())
 					}
 				}
-				err := nm.UpdateOriginalURL(currentNetworks[j].ChainID, networks[i].RPCURL, networks[i].FallbackURL)
+				err := nm.UpdateRpcURLs(currentNetworks[j].ChainID, networks[i].RPCURL, networks[i].OriginalRPCURL, networks[i].FallbackURL, networks[i].OriginalFallbackURL)
```

Comment:
why updating all urls here? we only need to update the original no?

Reply:
We're using those that are not "originals" and in case the user set env network-related vars, previously they were just updating "originals" but the RPC requests were made using "not originals", so that didn't have an effect. 
Also, this `Init` function is called only once on the app start and it's important to replace both, otherwise we're revealing our default app infura/pokt tokens.